apollo-client/src/test/java/com/ctrip/framework/apollo/spring/JavaConfigAnnotationTest.java

```diff
@@ -91,6 +105,44 @@ public void testApolloConfigWithInheritance() throws Exception {
     assertEquals(applicationConfig, bean.getSomeConfig());
   }
 
+
+  @Test
+  public void testEnableApolloConfigResolveExpressionSimple() {
+    mockConfig(ConfigConsts.NAMESPACE_APPLICATION, mock(Config.class));
+    mockConfig("xxx", mock(Config.class));
+    getSimpleBean(TestEnableApolloConfigResolveExpressionWithDefaultValueConfiguration.class);
```

Comment:
The current logic does prove the program working with no exception occurred, but it failed to verify namespace `xxx` is actually injected into Spring.
Here is some sample code I think might do the job, just for your reference.
```java
  @Test
  public void testEnableApolloConfigResolveExpressionSimple() {
    String someKey = "someKey";
    String someValue = "someValue";
    mockConfig(ConfigConsts.NAMESPACE_APPLICATION, mock(Config.class));
    Config xxxConfig = mock(Config.class);
    when(xxxConfig.getProperty(eq(someKey), anyString())).thenReturn(someValue);
    mockConfig("xxx", xxxConfig);
    TestEnableApolloConfigResolveExpressionWithDefaultValueConfiguration bean = getSimpleBean(TestEnableApolloConfigResolveExpressionWithDefaultValueConfiguration.class);
    assertEquals(someValue, bean.someKey);
    verify(xxxConfig, times(1)).getProperty(eq(someKey), anyString());
  }

  @Configuration
  @EnableApolloConfig(value = {ConfigConsts.NAMESPACE_APPLICATION, "${simple.namespace:xxx}"})
  protected static class TestEnableApolloConfigResolveExpressionWithDefaultValueConfiguration {
    @Value("${someKey}")
    private String someKey;
  }
```

Reply:
Thanks! New test add it, and test was passed.